### PR TITLE
test: Add log artifact for ginkgo node e2e and tune default ginkgo flags

### PR DESCRIPTION
**Pull request by bobbypage**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

This PR makes a few updates to the node e2e runner to make debugging the tests easier. Please review commit by commit.

The main changes are:

1. Emit a ginkgo log for each node. Currently when a node e2e image config has multiple images defined, the test is run in parallel for each image, and the results are all concatenated into a huge log file. Searching this log file becomes difficult, as some pod names/other artifacts are re-used across the tests on different hosts. As a result, to make debugging simpler, emit a log file for each node that contains just the ginkgo log that was run on that node.
2. Add some defaults flags for ginkgo. Specifically, `--no-color` to disable color as it makes examining the logs harder in prow and in text editors, and `-v` to enable verbose logs, so even if tests pass we receive the test output. This is helpful to compare passing runs to failed runs.   

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: test/e2e_node/runner/remote/run_remote.go [hunk 1]
```diff
@@ -359,7 +361,6 @@ func prepareGceImages() (*internalImageConfig, error) {
 				metadata:        getImageMetadata(metadata),
 				kernelArguments: imageConfig.KernelArguments,
 				machine:         imageConfig.Machine,
-				tests:           imageConfig.Tests,
```
Comment (SergeyKanzhelev) [inline test/e2e_node/runner/remote/run_remote.go:362]: thank you for removing this. Did you check we removed all the instances of this?

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

@xmcqueen sorry for stepping into your review. This change lgtm

File: test/e2e_node/runner/remote/run_remote.go (hunk 1 above)
Comment (bobbypage) [reply test/e2e_node/runner/remote/run_remote.go:362]: Yes, the only reference I could find is https://github.com/kubernetes/test-infra/pull/26017 but it was deleted in that PR.

**Pull request merged**